### ISSUE TITLE
Multipath critical for systems with internal drives.

### ISSUE THREAD

**Issue by Rad3d21** [aleex42/netapp-cdot-nagios#68]
Filers with internal drives only have 2 paths. The 4 path default is reporting critical because only 2 paths found.
(for example on Netapp 500F )
I would propose a 2 path default. (2 is also multi)

**Comment by aleex42** [comment]
Should be fixed, with @ebruening's commit, right?

**Comment by Rad3d21** [comment]
The 500F had a NSM8E module type. I see PSM8E in the code.

**Comment by ebruening** [comment]
It should be fine now. https://github.com/aleex42/netapp-cdot-nagios/pull/74

**Comment by Rad3d21** [comment]
Yes. That is now fine. However a new issue has arrived.... A90 has nsm16e shelf. That also needs to be added to the list with internal disk/2 paths.

**Comment by Rad3d21** [comment]
Yes that works ! Thank you.
